packages/stack/blockchain/package.json

```diff
@@ -49,7 +49,10 @@
       "../../../.eslintrc.json",
       "plugin:jest/recommended",
       "plugin:jest/style"
-    ]
+    ],
+    "rules": {
+      "jest/expect-expect": "off"
+    }
```

Comment:
We've introduced `jest`'s eslint extensions in the meanwhile so linting was failing because the blockchain tests don't have `expect()` assertions. This rule turns it off. No big deal, we might want to move to jest built-ins instead of using `assert` and `sinon`.